history/titles/d_tidemother.txt

```diff
@@ -1,25 +1,14 @@
-1.1.1 = {
+1.1.1={
 	liege=k_kul_tiras
+	law = cognatic_succession
+	law = succ_feudal_elective
 }
-24.1.1={holder=48170} #Finch[26002]
```

Comment:
Why the Stormsongs aren't religious leaders of Kul Tiras from the very beginning?
https://wow.gamepedia.com/Tidesages#History